assignments/week_3/exercise_3_3/exercise_3_3.md

```diff
@@ -0,0 +1,259 @@
+# Exercise 3.3
+Welcome to your first ROS exercise! We will first give a brief overview of what ROS is,
+then we will learn the basics of ROS nodes and learn how to launch nodes, and some commandline tools.
+
+## What is ROS?
+ROS stands for Robotic Operating System. It's a very convenient framework for programming robots.
+
+That was a short and probably not very useful explanation of what ROS is, but throughout
+the coming exercises we'll learn more about what ROS is by using its core features, and
+hopefully you'll gain a better understanding of what it is, but for now:
+
+## How does ROS work?
+The core of ROS deals with multiple small programs, called __nodes__, which communicate
+with each other. The important thing is that each node is only responsible for one aspect
+of the robot, which helps make the entire stack much cleaner.
+
+For example, we might have a
+nodes for identifying barrels, mapping, localization (finding out where the robot is) and
+path planning.
+
+## ROS Nodes and Launching Buzzsim
+Before we write our own node, let's first launch an existing node. Because the ROS ecosystem
+is quite large, we can easily use ROS nodes which other people have written for our own projects.
+
+First you'll need to build the simulator. Go to the Robojackets/Buzzsim/catkin_ws folder in your VM using a terminal. Then type catkin_make and hit enter. From a new terminal you can also type these commands in order:
+
+```bash
+cd RoboJackets/Buzzsim/catkin_ws
+catkin_make
+```
+This will build buzzsim, a simulation which we will use for many ROS exercises.
+
+Then we need to run 'roscore', still in catkin_ws:
+```bash
+roscore
+```
+
+Don't worry about what `roscore` is. You can think of it as something that handles all the ros functionality behind the
```

Comment:
This is covered in the video, so I think you can edit the line to just cover the common error message.